snapcraft/tests/sources/test_deb.py

```diff
@@ -60,3 +61,7 @@ def test_extract_and_keep_debfile(self):
 
         with open(deb_download, 'r') as deb_file:
             self.assertEqual('Test fake compressed file', deb_file.read())
+
+    def test_has_source_handler_entry_on_linux(self):
+        if sys.platform == 'linux':
```

Comment:
I find it nicer to use skip unless: https://docs.python.org/3/library/unittest.html#unittest.skipUnless